packages/eslint-config-react-native-community/index.js

```diff
@@ -30,13 +31,39 @@ module.exports = {
     },
   },
 
-  overrides: {
-    files: ['**/__tests__/**/*.js', '**/*.spec.js', '**/*.test.js'],
-    env: {
-      jest: true,
-      'jest/globals': true,
+  overrides: [
+    {
+      files: ['*.js'],
+      parser: 'babel-eslint',
+      plugins: ['flowtype'],
+      rules: {
+        // Flow Plugin
+        // The following rules are made available via `eslint-plugin-flowtype`
+
```

Comment:
unnecessary newline :D

Reply:
@thymikee just following the convention from rest of the file